bundles/org.openhab.io.openhabcloud/README.md

```diff
@@ -43,10 +43,13 @@ After installing this add-on, you will find configuration options in the openHAB
 
 ![Configuration](doc/configuration.png)
 
-Please note, that you should not expose all your items in this settings dialog.
-Also note that at present this feature is currently turned off.
-You just need to expose those items, which you want to be accessible by IFTTT.
-This setting has no affect on Alexa or Google Assistant functionality.
+By default both remote access and push notifications are enabled.  
+
+### Advanced Configuration
+
+For private hosted myopenHAB installations, the base URL can be changed to point another cloud instance.
+
+Private hosted myopenHAB installations may enable selected items in openHAB to have their state updates pushed to the cloud service for integrations with services like IFTTT.  Note that this is not supported on the community hosted myopenHAB service due to high load concerns and will have no effect if enabled with the default URL configured.  This is also not required for remote access through the cloud service to function.
```

Comment:
Can you reformat this to one sentence per line?

Reply:
Yes, good point